### PR TITLE
[container.requirements] Improve index entries for contiguous and reversible containers.

### PR DESCRIPTION
**Pull request by Eelis**
image

### CODE REVIEW

File: source/containers.tex [hunk 1]
```diff
@@ -366,10 +366,11 @@
 swap.
 
 \pnum
+\indextext{reversible container|see{container, reversible}}%
 If the iterator type of a container belongs to the bidirectional or
 random access iterator categories\iref{iterator.requirements},
 the container is called
-\term{reversible}
+\defnx{reversible}{container!reversible}
```
Comment (jensmaurer) [inline source/containers.tex:373]: Please use \defnadj{reversible}{container} and drop the "see" above (which comes for free now).

File: source/containers.tex [hunk 2]
```diff
@@ -465,8 +466,8 @@
 within that container.
 
 \pnum
-\indextext{container!contiguous}%
-A \defn{contiguous container}
+\indextext{contiguous container|see{container, contiguous}}%
+A \defnx{contiguous container}{container!contiguous}
```
Comment (jensmaurer) [inline source/containers.tex:470]: Same here: \defnadj, please

File: source/containers.tex (hunk 1 above)
Comment (Eelis) [reply source/containers.tex:373]: Hmm, using \defnadj would result in "reversible container" in the running text, but here we only want the word "reversible" to appear in the running text. \defnadjx also seems unsuitable for the same reason.

Maybe there ought to be a macro for defining an adjective while only making the adjective itself (and not the noun) appear in the running text?

Comment (jensmaurer) [reply source/containers.tex:373]: Ah, you're right. But that doesn't apply to the second change, below.

File: source/containers.tex (hunk 2 above)
Comment (Eelis) [reply source/containers.tex]: Done, thanks!

**Comment by Eelis** [comment]
Amended to use \defnadj.

**Pull request merged**